ivy/func_wrapper.py

```diff
@@ -1196,7 +1225,9 @@ def mini_helper(x):
 
         args = ivy.nested_map(args, mini_helper, include_derived=True)
         kwargs = ivy.nested_map(kwargs, mini_helper)
-        return fn(*args, **kwargs)
+        return (
+            ivy.astype(fn(*args, **kwargs), cast_to) if cast_to else fn(*args, **kwargs)
```

Comment:
we could just do a `ret = fn(*args, **kwargs)` on a previous line instead of calling `fn(*args, **kwargs)` in both conditions, thanks!

Reply:
Just asking for the sake of knowledge, does this make a difference in performance or just cleaner code?

Reply:
yeah just cleaner code 👍